consistent/consistent_test.go

```diff
@@ -0,0 +1,743 @@
+// Copyright (C) 2012-2014 Numerotron Inc.
+// Use of this source code is governed by an MIT-style license
+// that can be found in the LICENSE file.
+
+package consistent
+
+import (
+	"bufio"
+	"math/rand"
+	"os"
+	"runtime"
+	"sort"
+	"strconv"
+	"sync"
+	"testing"
+	"testing/quick"
+	"time"
+
+	. "github.com/thunderdb/ThunderDB/proto"
+	"github.com/thunderdb/ThunderDB/utils"
+)
+
+func NewNodeFromId(id string) Node {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
func NewNodeFromId should be NewNodeFromID